core-web/libs/edit-content/src/lib/components/dot-edit-content-sidebar/dot-edit-content-sidebar.component.ts

```diff
@@ -50,29 +50,29 @@ import { DotEditContentStore } from '../../store/edit-content.store';
     ]
 })
 export class DotEditContentSidebarComponent {
-    readonly store: InstanceType<typeof DotEditContentStore> = inject(DotEditContentStore);
-    readonly $identifier = this.store.getCurrentContentIdentifier;
-    readonly $formValues = this.store.formValues;
-    readonly $contentType = this.store.contentType;
-    readonly $contentlet = this.store.contentlet;
+    readonly $store: InstanceType<typeof DotEditContentStore> = inject(DotEditContentStore);
+    readonly $identifier = this.$store.getCurrentContentIdentifier;
+    readonly $formValues = this.$store.formValues;
+    readonly $contentType = this.$store.contentType;
+    readonly $contentlet = this.$store.contentlet;
 
     /**
      * Computed property that returns the workflow state of the content.
      */
     readonly $workflow = computed<DotWorkflowState | null>(() => ({
```

Comment:
@oidacra Why this computed could be `null`?

Reply:
I think can't be null.